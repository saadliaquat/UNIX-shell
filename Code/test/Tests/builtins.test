pwd
echo hehe
cd Tests
pwd
cd ..
echo hello bro
echo "testing 1 2 3"
echo "hello" "world" "hehe"