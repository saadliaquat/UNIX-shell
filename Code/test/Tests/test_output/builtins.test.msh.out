/home/os-fall-2023/code/test
hehe
/home/os-fall-2023/code/test/Tests
hello bro
testing 1 2 3
hello world hehe
